src/components/OurOffer/ComparisonTable/Table/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Typography } from "@mui/material";
+import { IFeature, IPlan, ITextProps } from "../../helpers";
+import classnames from "classnames";
+import CheckIcon from "@assets/icon/check-in-icon.svg?react";
+import CrossIcon from "@assets/icon/x-icon.svg?react";
+import useStyles from "./styles";
+
+interface IComparisonTableProps {
+  text: ITextProps;
+  features: IFeature[];
+  plans: IPlan[];
+}
+
+function Table({ features, plans }: IComparisonTableProps) {
+  const { classes } = useStyles();
+
+  function handleFeatureName(feature: IFeature) {
```

Comment:
Since it's rather simple, I would outsource the classnames logic to a variable and move the JSX together with the rest of the JSX at the end.